src/collections/_documentation/cli/releases.md

```diff
@@ -48,7 +48,7 @@ VERSION=`sentry-cli releases propose-version`
 
 ## Commit Integration {#sentry-cli-commit-integration}
 
-If you have [repositories configured]({%- link _documentation/workflow/releases.md -%}#link-a-repository) within your Sentry organization you can associate commits with your release.
+If you have [repositories configured]({%- link _documentation/workflow/releases/index.md -%}#link-a-repository) within your Sentry organization you can associate commits with your release.
 
 There are two modes in which you can use this. One is the fully automatic mode. If you are deploying from a git repository and sentry-cli can discover the git repository from the current working directory you can set the commits with the `--auto` flag:
```

Comment:
There are/there is constructions lead to dead writing. And "this" should be followed by a noun. 

But this para is confusing. I think it means to direct customers to the two modes, one of which is fully automatic. I'd suggest re-organizing to clearly identify the two modes, then create subhead that explains how to use the first mode, then the second mode.